From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Stefan Eissing <stefan@eissing.org>
Date: Mon, 8 Dec 2025 12:59:13 +0100
Subject: [PATCH 20/20] fix regression in
 77abeb39879fd13d220043641ece64110873fded the borked MDStapleOthers
 inheritance

Backported from mod_md v2.6.7 for Apache 2.4.66 (bundled mod_md v2.6.6)
---
 modules/md/mod_md_config.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules/md/mod_md_config.c b/modules/md/mod_md_config.c
index d6807c9..9688714 100644
--- a/modules/md/mod_md_config.c
+++ b/modules/md/mod_md_config.c
@@ -282,6 +282,7 @@ static void *md_config_merge(apr_pool_t *pool, void *basev, void *addv)
     nsc->profile = add->profile? add->profile : base->profile;
     nsc->profile_mandatory = (add->profile_mandatory != DEF_VAL)? add->profile_mandatory : base->profile_mandatory;
     nsc->stapling = (add->stapling != DEF_VAL)? add->stapling : base->stapling;
+    nsc->staple_others = (add->staple_others != DEF_VAL)? add->staple_others : base->staple_others;
     nsc->ari_renewals = (add->ari_renewals != DEF_VAL)? add->ari_renewals : base->ari_renewals;
     nsc->dns01_cmd = (add->dns01_cmd)? add->dns01_cmd : base->dns01_cmd;
     nsc->current = NULL;
